crates/objc2-encode/src/helper.rs

```diff
@@ -171,6 +172,26 @@ impl Primitive {
             Unknown => "?",
         }
     }
+
+    pub(crate) const fn size(self) -> usize {
+        match self {
+            // Numbers.
+            Self::Char | Self::UChar => 1,
+            Self::Short | Self::UShort => 2,
+            Self::Int | Self::UInt | Self::Float | Self::Bool => 4,
+            Self::Long | Self::ULong | Self::Double => 8,
+            Self::LongLong | Self::ULongLong | Self::LongDouble => 16,
```

Comment:
Yes, most probably. I've done all of this in a somewhat optimistic way. If you have a bit of documentation somewhere about it, it would be nice, or does it follow standard C in this regard?

Reply:
I think the best way would probably be something like:
```rust
Self::Char => core::mem::size_of::<core::ffi::c_char>(),
// ...
```

Though that conflicts with our MSRV of 1.60... I think we'll have to bump MSRV to 1.64 for this, though I'm probably fine with that (but it should happen in a separate PR, I'll try to get to doing that soon-ish).